rpcserver.go

```diff
@@ -3946,6 +3946,7 @@ func (r *rpcServer) dispatchPaymentIntent(
 			DestCustomRecords: payIntent.destCustomRecords,
 			DestFeatures:      payIntent.destFeatures,
 			PaymentAddr:       payIntent.paymentAddr,
+			MaxHtlcs:          1,
```

Comment:
Shouldn't this read the value from the intent? So we'll need to add these new fields in order to pass everything down from the RPC layer all the way to the router.

Reply:
Ah, or this is actually to force MPP only for the router sub-server.

Reply:
Yes indeed, disallow mpp on the main rpc interface